quelpa.el

```diff
@@ -1806,6 +1806,22 @@ the `quelpa' command has been run in the current Emacs session."
                 (quelpa item)))
             quelpa-cache))))
 
+;;;###autoload
+(defun quelpa-upgrade (rcp)
```

Comment:
Suggestion: name the variable `recipe`.  Abbreviations that have to be explained are often not worth it.  :)

Reply:
I've tried to keep the consistency with naming in this package. All other place recipe is referred as rcp. We can try to refactor it after this PR.